File: 설유민/Lv.1/12926-시저 암호/solution.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s, int n) {
    string answer = "";
    for (int i = 0; i < s.length(); i++) {
        if ((65 <= s[i]) && (s[i] <= 90)) {
            answer.push_back((s[i] + n - 65) % 26 + 65);
        }
        else if ((97 <= s[i]) && (s[i] <= 122)) {
            answer.push_back((s[i] + n - 97) % 26 + 97);
        }
        else {
            answer.push_back(s[i]);
        }
    }
    return answer;
}

File: 설유민/Lv.1/12926-시저 암호/solution.py
def solution(s, n):
    answer = ''
    for c in s:
        if 65 <= ord(c) <= 90:
            if 65 <= ord(c) + n <= 90:
                answer += chr(ord(c) + n)
            else:
                answer += chr(ord(c) + n - 26)
        elif 97 <= ord(c) <= 122:
            if 97 <= ord(c) + n <= 122:
                answer += chr(ord(c) + n)
            else:
                answer += chr(ord(c) + n - 26)
        else:
            answer += c
    return answer